php-practice/section13.php

```diff
@@ -144,6 +156,23 @@ function takeDice(Dice $ball) {
  * out something about them.
  */
 
+class GreatMovie {
+  public $greatMovie = 'Inception';
+}
+
+class BestMovie extends GreatMovie {
+  public $bestMovie = 'Star Wars Episode IV';
+}
+
+function pickMovie($greatMovie, $bestMovie) {
+  echo $greatMovie . 'isn\'t as good as' . $bestMovie;
+}
+
+$movieAnswer = new GreatMovie();
+
+/*$movieAnswer->pickMovie();*/
+
+echo SEPARATOR;
```

Comment:
I forgot to mention with this problem -- I'd like the `pickMovie` function to use *Typehinting* to enforce that the correct types of arguments are provided to the function. (Just like the `takeDice` function above.)